x/campaign/client/cli/tx_edit_campaign.go

```diff
@@ -10,11 +10,13 @@ import (
 	"github.com/tendermint/spn/x/campaign/types"
 )
 
-func CmdUpdateCampaignName() *cobra.Command {
+const flagName = "flagCampaignName"
```

Comment:
```suggestion
const flagName = "flag-campaign-name"
```

We should use kebab case for flag